tests/lpp_test.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jul 24 12:12:25 2020
+
+@author: smullally
```

Comment:
Probably unnecessary to put this in docstring. Git keeps track of the history for you. If we stop updating it, it is just going to be misleading.